flow/labels.py

```diff
@@ -11,16 +11,17 @@
 
 class label(object):
     """Decorate a :class:`~.FlowProject` class function as a label function.
-
+    
```

Comment:
@CryoZEUS Can remove these extra spaces on otherwise blank lines? I recommend configuring your editor to trim trailing whitespace when you save a file.

Reply:
Okay, will do that